### PR TITLE
Andrew/map

### PR DESCRIPTION
**Pull request by amillman**
@jonmchu @warrenshen 

- Implement maps feature, popups only display organization and name currently
- Add validation for checking a valid location if the user wants to be on the map

screen shot 2016-11-30 at 7 38 12 pm

### CODE REVIEW

File: app/assets/javascripts/components/registration_modal.es6.jsx [hunk 1]
```diff
@@ -39,14 +40,17 @@ class RegistrationModal extends React.Component {
     window.location = "/users/sign_in";
   }
 
-  _attemptRegistration(e) {
+  _attemptRegistration(response = null) {
+    var loc_id = null;
```
Comment (warrenshen) [inline app/assets/javascripts/components/registration_modal.es6.jsx]: Nit: lolol javascript is a camelCase language.

**Review by warrenshen — APPROVED** [review]
There's a gem https://github.com/alexreisner/geocoder
if you want more geolocation functionality

File: app/assets/javascripts/components/registration_modal.es6.jsx (hunk 1 above)
Comment (eric-ls) [reply app/assets/javascripts/components/registration_modal.es6.jsx]: gdi millman

Comment (amillman) [reply app/assets/javascripts/components/registration_modal.es6.jsx]: NOOOOOOOOOOOOOOOOOOOOO

**Comment by jonmchu** [comment]
LGTM

**Pull request merged**